spring-integration-gcp/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-gcp</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<name>Spring Integration GCP</name>
+	<description>Integration modules for Google Cloud Platform</description>
+	<artifactId>spring-integration-gcp</artifactId>
+	<organization>
```

Comment:
Yeah... What you suggest doesn't go under this `org.springframework.cloud` group umbrella and requires a separate maven management as long as it will be already a separate project even if in the same root directory...
No, am I over-panicking?